pkg/uhttp/body_print.go

```diff
@@ -0,0 +1,41 @@
+package uhttp
+
+// Implements a debugging facility for request responses. This changes
+// the behavior of `BaseHttpClient` with an unexported flag.
+//
+// If you always wanted to see the actual body of your response, now
+// you can 👁️👄👁️undress it🫦 to uncover all its... data!
+
+import (
+	"io"
+	"log"
+)
+
+type printReader struct {
+	reader io.Reader
+}
+
+func (pr *printReader) Read(p []byte) (int, error) {
+	n, err := pr.reader.Read(p)
+	if n > 0 {
+		log.Print(string(p[:n]))
```

Comment:
We shouldn't use `log.Print` here. If you want to write to stdout, you could use `fmt.Fprintf(os.Stdout, string(p[:n]))`